submit/workflow/test_new_submission.py

```diff
@@ -0,0 +1,26 @@
+"""Tests for workflow"""
+
+from unittest import TestCase, mock
+from submit import workflow
+from submit.workflow import processor
+
+
+class TestNewSubmissionWorkflow(TestCase):
+
+    def testVerifyUser(self):
+        seen = {}
+        submission = mock.MagicMock(submission_id=1234,
+                                    is_finalized=False,
+                                    submitter_contact_verified=False)        
+        nswfps = processor.WorkflowProcessor(
```

Comment:
This shows how the workflow is used.

Reply:
Test is helpful. Hopefully we'll have more complicated tests in future though this depends on what gets decided in terms of locking down workflow. I'm more in favor of allowing user to select order of steps they complete while disabling step transitions that are invalid (ie., process without uploading files)